packages/vite/package.json

```diff
@@ -32,9 +32,9 @@
     "./client": {
       "types": "./client.d.ts"
     },
-    "./runtime": {
-      "types": "./dist/node/runtime.d.ts",
-      "import": "./dist/node/runtime.js"
+    "./module-runner": {
```

Comment:
`module-runner` doesn't feel that right with the two words. I wonder if we should have something more generic here. Is there other things that may be imported by runtimes later on?
Should we keep it as:
```js
import { ModuleRunner } from 'vite/runtime'
```
with the meaning that it the entry point for Runtimes to import from?

Reply:
`module-runner` was something that you proposed 😄 

> Is there other things that may be imported by runtimes later on?

No, the idea is to keep this as small as possible. Making it generic would encourage putting more things here.

Reply:
I vote to `module-runner`. I'm fine with `runner`, too.

`runtime` feels a bit different to me. In the new environment API, I understand that the term "runtime" means the actual runtime (e.g. `node`, `deno`, `workerd`) and not the module runner/loader/executor built using Vite's API.